CHANGELOG.md

```diff
@@ -1,6 +1,117 @@
+* **New Data Source:** `vcd_nsxt_segment_qos_profile` to read NSX-T QoS Profiles ([#1120](https://github.com/vmware/terraform-provider-vcd/pull/1120))
+* **New Data Source:** `vcd_nsxt_segment_security_profile` to read NSX-T Segment Security Profiles ([#1120](https://github.com/vmware/terraform-provider-vcd/pull/1120))
+* **New Resource:** `vcd_nsxt_segment_profile_template` to manage NSX-T Segment Profile Templates ([#1120](https://github.com/vmware/terraform-provider-vcd/pull/1120))
+* **New Data Source:** `vcd_nsxt_segment_profile_template` to read NSX-T Segment Profile Templates ([#1120](https://github.com/vmware/terraform-provider-vcd/pull/1120))
+* **New Resource:** `vcd_nsxt_global_default_segment_profile_template` to manage NSX-T Global Default Segment Profile Templates ([#1120](https://github.com/vmware/terraform-provider-vcd/pull/1120))
+* **New Data Source:** `vcd_nsxt_global_default_segment_profile_template` to read NSX-T Global Default Segment Profile Templates ([#1120](https://github.com/vmware/terraform-provider-vcd/pull/1120))
+* **New Resource:** `vcd_org_vdc_nsxt_network_profile` to manage default Segment Profile Templates for NSX-T VDCs ([#1120](https://github.com/vmware/terraform-provider-vcd/pull/1120))
+* **New Data Source:** `vcd_org_vdc_nsxt_network_profile` to read default Segment Profile Templates for NSX-T VDCs ([#1120](https://github.com/vmware/terraform-provider-vcd/pull/1120))
+* **New Resource:** `vcd_nsxt_network_segment_profile` to manage individual Segment Profiles or Segment Profile Templates for NSX-T Org VDC Networks ([#1120](https://github.com/vmware/terraform-provider-vcd/pull/1120))
+* **New Data Source:** `vcd_nsxt_network_segment_profile` to read individual Segment Profiles or Segment Profile Templates for NSX-T Org VDC Networks ([#1120](https://github.com/vmware/terraform-provider-vcd/pull/1120))
+* **New Resource:** `vcd_nsxt_edgegateway_l2_vpn_tunnel` to manage Edge Gateway L2 VPN Tunnel sessions ([#1061](https://github.com/vmware/terraform-provider-vcd/pull/1061))
+* **New Data Source:** `vcd_nsxt_edgegateway_l2_vpn_tunnel` to read Edge Gateway L2 VPN Tunnel sessions ([#1061](https://github.com/vmware/terraform-provider-vcd/pull/1061))
+* **New Resource:** `vcd_nsxt_edgegateway_dns` to manage Edge Gateway DNS forwarder configuration ([#1154](https://github.com/vmware/terraform-provider-vcd/pull/1154))
+* **New Data Source:** `vcd_nsxt_edgegateway_dns` to read Edge Gateway DNS forwarder configuration ([#1154](https://github.com/vmware/terraform-provider-vcd/pull/1154))
+* **New Resource:** `vcd_vm_vgpu_policy` to manage VM vGPU compute policy configuration ([#1167](https://github.com/vmware/terraform-provider-vcd/pull/1167))
+* **New Data Source:** `vcd_vm_vgpu_policy` to read VM vGPU compute policies ([#1167](https://github.com/vmware/terraform-provider-vcd/pull/1167))
```

Comment:
Shouldn't these be in experimental section? @adezxc

Reply:
Yes, it seems they are duplicated. Removed the one entry from FEATURES list.